week07/seoayeong-crud-api/src/main/java/practice/seoayeongcrudapi/member/service/MemberService.java

```diff
@@ -0,0 +1,51 @@
+package practice.seoayeongcrudapi.member.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import practice.seoayeongcrudapi.member.controller.dto.request.CreateMemberRequest;
+import practice.seoayeongcrudapi.member.controller.dto.request.UpdateMemberRequest;
+import practice.seoayeongcrudapi.member.controller.dto.response.MemberResponse;
+import practice.seoayeongcrudapi.member.entity.Member;
+import practice.seoayeongcrudapi.member.repository.MemberRepository;
+
+@Service
+@RequiredArgsConstructor
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+
+//    public void signUp(CreateMemberRequest createMemberRequest) {
+//        Member member = createMemberRequest.toEntity();
+//        memberRepository.save(member);
+//    }
+//
+//    public MemberResponse getMember(String name) {
+//        Member member = memberRepository.findByName(name);
+//        return MemberResponse.from(member);
+//    }
+
+    public void signUp(CreateMemberRequest createMemberRequest) {
+        Member member = createMemberRequest.toEntity();
+        memberRepository.save(member);
+    }
+
+    public MemberResponse getMemberByName(Long id) {
```

Comment:
메소드명 변경 해주셔야할 것 같아요! 현재 id로 member를 찾고있는 것 같습니다
```suggestion
    public MemberResponse getMemberById(Long id) {
```